compiler/qsc_doc_gen/src/display.rs

```diff
@@ -615,12 +615,12 @@ where
     write!(formatter, "}}")
 }
 
-fn display_type_params(generics: &[GenericParam]) -> String {
+fn display_type_params(generics: &[HirTypeParameter]) -> String {
     let type_params = generics
         .iter()
         .filter_map(|generic| match generic {
-            GenericParam::Ty(name) => Some(name.name.clone()),
-            GenericParam::Functor(_) => None,
+            HirTypeParameter::Ty { name, .. } => Some(name.clone()),
```

Comment:
I think it's worth filing a follow up to make sure docs generation includes type bounds. Right now it's not critical since the stdlib doesn't have anything with bounds, but the generated docs for your new library (other PR) will be wrong in VS Code until this is updated.

Reply:
Went ahead and added a printout of bounds here using `ClassConstraints`' `Display` impl.